lib/src/importer/base.dart

```diff
@@ -28,18 +26,10 @@ abstract class ImporterBase extends sass.Importer {
     try {
       parsedUrl = Uri.parse(url);
     } on FormatException catch (error) {
-      sendAndThrow(paramsError("$field is invalid: $error"));
+      throw "$field is invalid: $error";
```

Comment:
If we're going to make the compiler officially handle these errors as well (which I agree makes sense, since there's not really a better way for the host to recover even if it does detect them), we should update the messages to be more meaningful to the end user. The current error messages are targeted at host authors, so they include the protocol fields, but for a user this should probably just say "Invalid URL".

Reply:
Agreed. Will check what dart-sass throw and update the error message to a similar one.